src/components/Count.tsx

```diff
@@ -33,19 +41,32 @@ class Count extends Component<CountProps> {
   }
 }
 
-const mapStateToProps = (state: { count: number }) => {
+interface mapState {
+  userReducer: {
+    displayName: string;
+    email: string;
+    uid: string;
+  };
+  countReducer: {
+    count: number;
+  };
+}
+const mapStateToProps = (state: mapState) => {
   return {
-    count: state.count,
+    displayName: state.userReducer.displayName,
+    email: state.userReducer.email,
+    uid: state.userReducer.uid,
+    count: state.countReducer.count,

```

Comment:
If these aren't being used by the component, how come they're being injected into it?